src/applications/discharge-wizard/components/FormQuestions.jsx

```diff
@@ -4,7 +4,6 @@ import PropTypes from 'prop-types';
 
 // Relative Imports
 import { focusElement, getScrollOptions } from 'platform/utilities/ui';
-import recordEvent from 'platform/monitoring/record-event';
```

Comment:
Design system web components come with analytics baked in. Per this thread: https://dsva.slack.com/archives/CKCRMLGAC/p1709663267129869?thread_ts=1709653964.294149&cid=CKCRMLGAC we will be removing custom events from our components.